[Observability Docs] add user-specific prompts docs (#1705)
elastic/docs-content: solutions/observability/observability-ai-assistant.md

    Ask something to the AI Assistant related with the indexed data.

### Add user-specific system prompts

User-specific prompts customize how the AI assistant responds by appending personalized instructions to built-in system prompts. For example, you could specify "Always respond in French," and all subsequent responses will be in French.

A user-specific prompt only applies to the user that sets it.

To edit the **User-specific System Prompt**:

> 1. To open AI Assistant settings, find `AI Assistants` in the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md).
> 2. Under **{{obs-ai-assistant}}**, click **Manage settings**.

Reviewer: ```suggestion
1. Go to the **{{obs-ai-assistant}}** management page. You can find it in the **Management** menu or by using the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md).
```

    Ask something to the AI Assistant related with the indexed data.

### Add user-specific system prompts

> User-specific prompts customize how the AI assistant responds by appending personalized instructions to all of your system prompts. For example, you could specify "Always respond in French," and all subsequent responses will be in French.

Reviewer 2: Perhaps we should make it clear that user instructions are appended to the built-in prompts.

```suggestion
User-specific prompts customize how the AI assistant responds by appending personalized instructions to built-in system prompts. For example, you could specify "Always respond in French," and all subsequent responses will be in French.
```

A user-specific prompt only applies to the user that sets it.

> % Do we have any other suggestions on how this could be useful for users?

Reviewer 2: Instructions are very useful if they want to configure specific workflows. I think @davidgeorgehope had some concrete examples
